Lombiq.Vsix.Orchard/Commands/OpenErrorLogCommand.cs

```diff
@@ -91,25 +91,32 @@ private async Task InitalizeWatchersAsync()
                 watcher.LogUpdated += LogFileUpdatedCallback;
             }
 
-            var settings = await _logWatcherSettingsAccessor.GetSettingsAsync();
+            var settings = await _logWatcherSettingsAccessor.GetSettingsAsync().ConfigureAwait(true);
             settings.SettingsUpdated += LogWatcherSettingsUpdatedCallback;
 
             if (settings.LogWatcherEnabled) StartLogFileWatching();
         }
 
         private void OpenErrorLogCommandBeforeQueryStatusCallback(object sender, EventArgs e) =>
-            _ = UpdateOpenErrorLogCommandAccessibilityAndTextAsync();
+#pragma warning disable VSTHRD102 // Implement internal logic asynchronously

```

Comment:
Mindig, minden suppresst meg kell indokolni.

Reply:
Nem azt mondtam, hogy át kell őket emelni attribute-ba :).

Reply:
Gondoltam, hogy így jobban látszik a Justification. írjam vissza #pragma formátumra?

Reply:
Jobban belegondolva nem, igazad van, mivel itt akárhány async használat is lesz ezekben a metódusokban, azt ugyanúgy kell csinálni, indokolt a metódus-szintű attribute-os suppress.

A teljes namespace ismétlése ezeknél is probléma viszont.